libs/geoservices/covid/src/lib/pages/maps/components/time-map/time-map.component.html

```diff
@@ -0,0 +1,50 @@
+<div id="map-section">
+  <tamu-gisc-drawer [width]="'20rem'" [position]="'left'">
+    <div class="content">
+      <h3>Map Controls</h3>
+      <input type="checkbox" name="county-state-toggle" (click)="stateToggle()">
+      <label for="county-state-toggle">Counties/State</label>
```

Comment:
Feel free to use `<tamu-gisc-checkbox>` component from `UIFormsModule`.

Reply:
done